docs/UserGuide.md

```diff
@@ -277,14 +283,34 @@ Format: `list-t`
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
 A task can have any number of tags (including 0)
 </div>
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
+A task can also be assigned to multiple persons (including 0)
+</div>
```

Comment:
Should the 2 tips be under one div class instead of a separate one?

Reply:
okay! wasn't sure about the standardisation since iirc this is the only spot with 2 tips. Changed, thanks!